website-gatsby/templates/index.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import {Home} from 'ocular-gatsby/components';
+import './style.css';
+
+if (typeof window !== 'undefined') {
+  window.website = true;
+}
+
+const HeroExample = 'div';
+
+// require('./example-gltf').default;
+
+export default class IndexPage extends React.Component {
+  render() {
+    return <Home HeroExample={HeroExample} />;
```

Comment:
kepler.gl home page has plenty of js react component that does fancy fade in and fade out base don scroll. Does this mean we can still keep our current `Home` [component](https://github.com/keplergl/kepler.gl/blob/master/website/src/components/home.js)?

Reply:
+1

Reply:
>  Does this mean we can still keep our current Home component?

Yes of course. Gatsby is 100% React based and is designed to let you wire up any React rendered components and render them as separate static HTML+JS pages. 

I have now wired up gatsby with redux and initial webpack settings but more work is needed. However it turns out that kepler has **a lot** of special build settings etc, it was too much for me to handle in just 2 hours.

@macrigiuseppe @heshan0131  can we collaborate on finishing up the home page component integration?

Just go to `website-gatsby`, uncomment the import in `templates/index.js`  do `yarn` and `yarn start` and start fixing errors :)

If we do it together I think we can get it done quickly...